tests/autoyast/installation.pm

```diff
@@ -343,7 +343,7 @@ sub run {
     $self->wait_boot if check_var('BACKEND', 'ipmi') and not get_var('VIRT_AUTOTEST') and not $pxe_boot_done;
 
     # Second stage starts here
-    $maxtime = 1000;
+    $maxtime = 1000 * get_var('TIMEOUT_SCALE', 1);    # Max waiting time for stage 2
```

Comment:
Why not `bmwqemu::scale_timeout(1000)`?

Reply:
Simply because the same kind of code is already used in other parts in this file, and I first don't wanted to modify all (mainly to be sure to not break anything :D).
But I can modify all lines.